lib/rust/ensogl/example/profiling-run-graph/src/lib.rs

```diff
@@ -1,4 +1,6 @@
-//! Demo scene showing a sample flame graph.
+//! Demo scene showing a sample flame graph. Can be used to display a log file, if you have one.
+//! To do so, set the `PROFILER_LOG_DATA` to contain the profiling log via `include_str`, and it
```

Comment:
This will not work. An environment variable cannot exceed 32KB, and debug-level profiles routinely exceed 20MB. But also, why set a profile at compile time? We currently render a file obtained at runtime from `dist/content/proflie.json`

Reply:
Loading the data at runtime is a bit more useful, just a bot more work. But I changed to do load the data instead of hard-coding the content..